scalableminds/webknossos
# Allow Overwriting Missing Datasets From Different Datastore

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up two datastores (e.g. localhost and a standalone), mark one as scratch
- move a dataset between the two, refresh dataset list
- dataset should be reported wherever it is
- if both have the dataset, the non-scratch one should take precedence over scratch
- if two non-scratch datasets have the dataset, the first one that reported it keeps it

### Issues:
- fixes #4458 

------
